sql/show.sh

```diff
@@ -86,15 +86,15 @@ echo ''
 echo '###### SHOW COMMAND TEST-0002 ######'
 echo '###### Status RUNNING  ######'
 init_catalog
-pg_rman backup -B ${BACKUP_PATH} -b full -Z -p ${TEST_PGPORT} -d postgres --quiet &
+pg_rman backup -B ${BACKUP_PATH} -b full -Z -p ${TEST_PGPORT} -d postgres --quiet & # run in background
 sleep 1
 pg_rman show -B ${BACKUP_PATH} > ${TEST_BASE}/TEST-0002-show.out 2>&1
 if grep "RUNNING" ${TEST_BASE}/TEST-0002-show.out > /dev/null ; then
      echo 'OK: RUNNING status is shown properly.'
 else
      echo 'NG: RUNNING status is not shown.'
 fi
-sleep 5
```

Comment:
I think you need to rebase the branch to the master because this change don't related to support v15.